src/elements/badge.svelte

```diff
@@ -25,3 +22,7 @@ addStyles();
 >
   {label}
 </small>
+
+<style>
+@import './../../prime.css';
```

Comment:
[nit] does `./..` === `../`?

```suggestion
@import '../../prime.css';
```